lib/s390base.pm

```diff
@@ -37,13 +49,29 @@ sub copy_testsuite {
     save_screenshot;
 }
 
+=head2 execute_script
+
+ execute_script();
+
+Execute script and upload script logs
+
+=cut
 sub execute_script {
     my ($self, $script, $scriptargs, $timeout) = @_;
```

Comment:
There are parameters here `$script, $scriptargs, $timeout`, this needs to be mentioned in the documentation.